### PR TITLE
[ENH]: Add support for masks (GM/WM/etc)

### PR DESCRIPTION
**Pull request by fraimondo**
### Are you requiring a new dataset or marker?

- [X] I understand this is not a marker or dataset request

### Which feature do you want to include?

In the same way we support atlases and coordinates, we should be able to support masks to be used, for example, in the spheres masker.

### How do you imagine this integrated in junifer?

Copy/paste the atlases and modify for masks (give or take the family and atlas-specific parameters)

### Do you have a sample code that implements this outside of junifer?

_No response_

### Anything else to say?

a "good" general purpose GM mask :)
https://zenodo.org/record/6463123#.YyHOOrRBzEY

### CODE REVIEW

File: junifer/data/utils.py
Comment (synchon) [inline junifer/data/utils.py:19]: list of float or int, or np.ndarray?

File: junifer/markers/crossparcellation_functional_connectivity.py
Comment (synchon) [inline junifer/markers/crossparcellation_functional_connectivity.py:37]: `(default None).`

File: junifer/markers/functional_connectivity_parcels.py
Comment (synchon) [inline junifer/markers/functional_connectivity_parcels.py:45]: `(default None).`

File: junifer/markers/functional_connectivity_spheres.py
Comment (synchon) [inline junifer/markers/functional_connectivity_spheres.py:48]: `(default None).`

File: junifer/markers/parcel_aggregation.py
Comment (synchon) [inline junifer/markers/parcel_aggregation.py:40]: `(default None).`

File: junifer/markers/sphere_aggregation.py
Comment (synchon) [inline junifer/markers/sphere_aggregation.py:42]: `(default None).`

File: junifer/data/masks.py
Comment (fraimondo) [reply junifer/data/masks.py:28]: not entirely. Inner values could be anything

Comment (synchon) [reply junifer/data/masks.py]: But shouldn't be the inner dict keys be string?

Comment (synchon) [reply junifer/data/masks.py]: Okay now, saw the change.

File: junifer/data/tests/test_data_utils.py
Comment (synchon) [inline junifer/data/tests/test_data_utils.py:35]: `list of float`

**Review by synchon — CHANGES_REQUESTED** [review]

**Comment by synchon** [comment]
Will wait for the CI to complete before merging.

**Pull request merged**
